src/modules/inspect/inspect-correlation/components/inspect-panel/components/process-instance-list/process-instance-list.ts

```diff
@@ -0,0 +1,260 @@
+      this.sortSettings.ascending = !this.sortSettings.ascending;
+
+      this.sortedTableData = this.sortList(this.sortSettings.sortProperty);
+    } else {
+      this.sortedTableData = this.tableData;
+    }
+
+    const processInstanceToSelectExists: boolean = this.processInstanceToSelect !== undefined;
+    if (processInstanceToSelectExists) {
+      const instanceAlreadyExistInList: ProcessInstanceTableEntry = this.sortedTableData.find(
+        (processInstance: ProcessInstanceTableEntry) => {
+          return processInstance.processInstanceId === this.processInstanceToSelect.processInstanceId;
+        },
+      );
+
+      if (instanceAlreadyExistInList) {
+        this.processInstanceToSelectTableEntry = undefined;
+      } else {
+        const processInstanceToSelectTableEntry: Array<
+          ProcessInstanceTableEntry
+        > = this.convertProcessInstancesIntoTableData([this.processInstanceToSelect]);
+
+        this.processInstanceToSelectTableEntry = processInstanceToSelectTableEntry[0];
+        this.selectProcessInstance(this.processInstanceToSelectTableEntry);
+      }
+
+      this.processInstanceToSelect = undefined;
+    }
+
+    this.paginationShowsLoading = false;
+  }
+
+  public pageSizeChanged(newValue, oldValue): void {
+    const isNotInitializedYet = oldValue === undefined;
+    if (isNotInitializedYet) {
+      return;
+    }
+
+    const showAllProcessInstances: boolean = this.pageSize === 0;
+    if (showAllProcessInstances) {
+      this.currentPage = 1;
+    }
+
+    const isFirstPage: boolean = this.currentPage === 1;
+    if (isFirstPage) {
+      const payload = {
+        offset: 0,
+        limit: this.pageSize,
+      };
+      this.eventAggregator.publish(environment.events.inspectCorrelation.updateProcessInstances, payload);
+
+      return;
+    }
+
+    const currentOffset: number = (this.currentPage - 1) * oldValue;
+
+    this.currentPage = Math.floor(currentOffset / this.pageSize) + 1;
+  }
+
+  public currentPageChanged(newValue: number, oldValue: number): void {
+    const isNotInitializedYet = oldValue === undefined;
+    if (isNotInitializedYet) {
+      return;
+    }
+
+    const payload = {
+      offset: this.currentPage === 1 || this.currentPage === 0 ? 0 : (this.currentPage - 1) * this.pageSize,
+      limit: this.pageSize,
+    };
+
+    this.eventAggregator.publish(environment.events.inspectCorrelation.updateProcessInstances, payload);
+  }
+
+  private convertProcessInstancesIntoTableData(
+    processInstances: Array<DataModels.Correlations.ProcessInstance>,
+  ): Array<ProcessInstanceTableEntry> {
+    return processInstances.map((processInstance: DataModels.Correlations.ProcessInstance) => {
+      const formattedStartedDate: string = getBeautifiedDate(processInstance.createdAt);
+
+      const tableEntry: ProcessInstanceTableEntry = {
+        startedAt: formattedStartedDate,
+        state: processInstance.state,
+        user: 'Not supported yet.',
```

Comment:
Man könnte hier die userId aus der Identity verwenden. 

```suggestion
        user: processInstance.identity.userId,
```